hermes-common/src/main/java/pl/allegro/tech/hermes/common/hook/HooksHandler.java

```diff
@@ -21,24 +23,35 @@ public void addShutdownHook(ServiceAwareHook hook) {
     }
 
     public void shutdown(ServiceLocator serviceLocator) {
+        deregisterGlobalShutdownHook();
```

Comment:
@wojtkiewicz from now on you can disable global shutdown hook via hermes builder. It is disabled in integration & benchmark tests